tests/rptest/services/redpanda.py

```diff
@@ -1657,14 +1662,29 @@ def __init__(self,
         # later to dataclass
         self._cc_config = context.globals[self.GLOBAL_CLOUD_CLUSTER_CONFIG]
 
-        self._provider_config = {
-            "access_key":
-            context.globals.get(SISettings.GLOBAL_S3_ACCESS_KEY, None),
-            "secret_key":
-            context.globals.get(SISettings.GLOBAL_S3_SECRET_KEY, None),
-            "region":
-            context.globals.get(SISettings.GLOBAL_S3_REGION_KEY, None)
-        }
+        self._provider_config = {}
+        match get_cloud_provider():
```

Comment:
There is no "gcp" case, doesn't this break it for GCP?

Reply:
I think I mistakenly assumed that the GCP code wouldn't use S3 settings, but looking at how the code used to flow vs now I think you're totally right. Will fix this.